README.md

```diff
@@ -80,7 +80,7 @@ event NewOptionType(
         address indexed underlyingAsset,
         uint96 exerciseAmount,
         uint96 underlyingAmount,
-        uint40 exerciseTimestamp,
+        uint40 earliestExerciseTimestamp,
```

Comment:
Why not? Someone will look at this field and instantly know what it means, vs. if it's 'exerciseTimestamp' having to reason about it, consult the docs, potentially forget and have to re-look it up, confuse it with the only time on which it can be exercised thinking that's how to do European options, etc.

Reply:
Reverting back